src/utils.js

```diff
@@ -349,9 +349,30 @@ const generateDetailTable = (metrics, rowData, {
 	return table;
 };
 
+const generateGui = (table) => {
+	const output = {
+		metrics: [],
+		repos: [],
+		rows: [],
+	};
+
+	for (let i = 0; i < table.length; i++) {
+		output.rows.push(table[i.toString()]
```

Comment:
```suggestion
		output.rows.push(table[i]
```
number properties are implicitly toStringed